spec/storage/string_interpolation/test.txt

```diff
@@ -0,0 +1 @@
+L#9U"gAc-XLA*/Ac|1M:
```

Comment:
Could you please add a description for this file's purpose that it doesn't get lost?